### PR TITLE
fix transformation for xgboost model

### PR DESCRIPTION
**Pull request by shanghuiyang**
This patch fixes the transformation while loading an xgboost model file. When the model file is saved from an xgboost model with param "objective= 'multi:softprob'" or "objective= 'multi:softmax'", the loaded model should use TransformSoftmax rather than TransformRaw. This patch works well on my xgboost model.

### CODE REVIEW

**Comment by nikolaydubina** [comment]
cc: @dmitryikh
